lib/BNO/BNO.cpp

```diff
@@ -7,6 +7,7 @@ Bno::Bno()
   yaw = 0;
   target_yaw = 0;
   current_yaw = 0;
+  isRightFlag=false;
 }
```

Comment:
follow some standard if you want to use `camelCase` use it or `low_casing` but dont mix them